src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -104,6 +104,9 @@ static bool mpuSpi6000InitDone = false;
 
 static IO_t mpuSpi6000CsPin = IO_NONE;
 
+#define SPI_CLOCK_MPU6000_INITIALIZATION SPI_CLOCK_INITIALIZATON
+#define SPI_CLOCK_MPU6000_FAST           SPI_CLOCK_FAST
```

Comment:
The constants may be incorrectly named, but if so they should be fixed in a separate PR. PRs, especially bug fixes, should be atomic - that is they should do one thing only.

Having the name change makes this PR more difficult to review, since it is difficult to see the essence of what is changes. Please update the PR by removing these two #defines and updating the rest of the code accordingly.

A separate PR that changes the names of the clock constants would be welcome. But if your objection is to the use of the term FAST, then `SPI_CLOCK_MPU6000_FAST` is hardly better than `SPI_CLOCK_FAST` and doesn't really remove the confusion you mentioned.